### ISSUE TITLE
Reg\Events - Internal User can view a transfer

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1713]
#### Description:
As an internal user, I am able to view the details of a transfer that was reported by an external user

#### Acceptance Criteria:
Given that I am on the Transfer Requests grid
When I click view details on a row
Then I am brought to the page with the event form that was filled,in view only mode, so that I can see the event that was reported

[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=13060-6973&t=lsl3ZaYZigOiKk3a-4) 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by patriciarussellCAS** [comment]
Hey team, is this already done through other Transfer tickets? I believe we can already see transfer information as an internal user, correct? 
@zoeyli-46 or @andrea-williams ??

**Comment by zoeyli-46** [comment]
Closing this: this was created when we were still under the assumption that external users would be filing the transfer!
